src/components/existing-request/nr-approved-gray-box.vue

```diff
@@ -119,13 +119,13 @@
             <v-icon small>
               mdi-phone
             </v-icon>&nbsp;Canada and U.S. Toll Free:
-            <a href="tel:+1877-370-1033">1-877-370-1033</a>
+            <a href="tel:+1877-526-1526">1-877-526-1526</a>
           </p>
           <p>
             <v-icon small>
               mdi-phone
             </v-icon>&nbsp;Victoria Office:
-            <a href="tel:250-370-1033">250-370-1033</a>
+            <a href="tel:250-387-7848">250-387-7848</a>
```

Comment:
This gray box, and the numbered company bullets below, aren't unique to alteration NRs. Please confirm with Mihai or Janis that it's OK to update the phone number for all NRs.

Reply:
No, this is unique to alteration NRs.

Reply:
I'm sorry --  you are right about this component.

However, the code in numbered-company-bullets.vue appears to be for conversions only, so maybe you don't need to change the phone numbers there?

Reply:
Conversion aka alteration 😸 so both need the change.

Reply:
Oh, I keep forgetting that. You're right again! Resolving.